cmd/signature-v4-parser.go

```diff
@@ -69,9 +69,6 @@ func parseCredentialHeader(credElement string) (credentialHeader, APIErrorCode)
 	if e != nil {
 		return credentialHeader{}, ErrMalformedCredentialDate
 	}
-	if credElements[2] == "" {
-		return credentialHeader{}, ErrMalformedCredentialRegion
```

Comment:
For example this situation we can agree that we have to relax and we will never send malformed credential anymore.